lib/mixlib/install/product.rb

```diff
@@ -144,30 +145,35 @@ def lookup(key, version = :latest)
   # Products in alphabetical order
 
   product "analytics" do
-    product_name "Analytics Platform"
-    package_name "opscode-analytics"
-    ctl_command "opscode-analytics-ctl"
-    config_file "/etc/opscode-analytics/opscode-analytics.rb"
+    product_name    "Analytics Platform"
+    package_name    "opscode-analytics"
+    omnibus_project "opscode-analytics"
```

Comment:
I think instead of adding `omnibus_project` as a standard property we can do something like below:

```
def omnibus_project(value = nil)
  if value.nil?
    @omnibus_project || @package_name
  else
    @omnibus_project = value
  end
```